virt/support/virt-support-tools.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="virt-support-tools"]
+= {VirtProductName} support tools
+include::_attributes/common-attributes.adoc[]
+:context: virt-support-tools
+
+toc::[]
+
+{VirtProductName} provides tools for collecting data about your environment, monitoring your cluster and virtual machines (VMs), and troubleshooting.
+
+{product-title} web console::
+* *Overview* page: Cluster details, status, alerts, inventory, and resource usage charts.
+* *Virtualization* page:
+** xref:../../virt/virt-web-console-overview.adoc#virtualization-overview-overview_virt-web-console-overview[*Overview* tab]: {VirtProductName} resources, usage, alerts, and status.
```

Comment:
We might want to re-work some of these lists/definitions so that they have more parallel construction. I also think this section both looks incomplete _and_ is a bit visually busy with all of the bullet points (even though I usually love a bulleted list!). I'm curious if we could use definition lists for some of them. Or maybe even tables?!